packages/admin-server/src/app.ts

```diff
@@ -191,6 +191,22 @@ export const createApp = async (): Promise<{app: Koa, config: Config}> => {
     });
   });
 
+  // hub
+  rootRouter.get('/hub', oidcCtrl.loggedIn, async ctx => {
```

Comment:
The path should be `/g` and `/g/:path`


So that 

`https://hub.celu.aws.primehub.io/console/g/*` would go to the new page.